panel/widgets/chat.py

```diff
@@ -800,20 +806,20 @@ def _replace_placeholder(self, entry: Optional[ChatEntry] = None) -> None:
 
     def _build_entry(
             self,
-            value: Union[ChatEntry, dict, Any],
-            user: Optional[str] = None,
-            avatar: Optional[Union[str, BinaryIO]] = None,
-        ) -> ChatEntry:
+            value: ChatEntry |  dict | Any,
+            user: str | None = None,
+            avatar: str | BinaryIO | None = None,
+        ) -> ChatEntry | None:
```

Comment:
What's the earliest Python version Panel supports? If it's 3.9, I don't think `|` is supported yet:
https://peps.python.org/pep-0604/

Reply:
Nevermind, Philipp mentioned from future import annotations should allow this.